profiles/default.yml

```diff
@@ -12,8 +12,6 @@ charts:
       lines:
```

Comment:
Since `Vector` has one entry, can you remove `lines`?

Reply:
The parser is stupid, it expects the format to be the same even in the case of one line.